[CI] SamplingServiceTests testMaybeSample failing

**Build Scans:**
- [elasticsearch-intake #31044 / part1](https://gradle-enterprise.elastic.co/s/gkdoih6rpihng)
- [elasticsearch-pull-request #103282 / part-1](https://gradle-enterprise.elastic.co/s/267qcaycwftuk)
- [elasticsearch-pull-request #103305 / part-1](https://gradle-enterprise.elastic.co/s/l6dcognzhe5wa)
- [elasticsearch-pull-request #103282 / part-1](https://gradle-enterprise.elastic.co/s/wmcwc72sbs6ua)
- [elasticsearch-pull-request #103272 / part-1](https://gradle-enterprise.elastic.co/s/uqdtfxckcjo6e)
- [elasticsearch-pull-request #103271 / part-1](https://gradle-enterprise.elastic.co/s/kkzvcmmsu2kdm)
- [elasticsearch-pull-request #103263 / part-1](https://gradle-enterprise.elastic.co/s/vgmz7eltvha5w)
- [elasticsearch-pull-request #103246 / part-1](https://gradle-enterprise.elastic.co/s/paku2tgkkzl6q)
- [elasticsearch-pull-request #103244 / part-1](https://gradle-enterprise.elastic.co/s/lsfyxggyxx5p6)
- [elasticsearch-pull-request #103229 / part-1-fips](https://gradle-enterprise.elastic.co/s/an2vzi7am2gfo)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.ingest.SamplingServiceTests.testMaybeSample" -Dtests.seed=9B3BD665D1139F8D -Dtests.locale=hi-Deva-IN -Dtests.timezone=Etc/GMT0 -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.ingest.SamplingServiceTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testMaybeSample'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <69>
     but: was <51>
```

**Issue Reasons:**
- [main] 11 failures in test testMaybeSample (6.5% fail rate in 169 executions)
- [main] 9 failures in step part-1 (8.0% fail rate in 113 executions)
- [main] 9 failures in pipeline elasticsearch-pull-request (8.2% fail rate in 110 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -472,9 +472,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.external.http.sender.RequestExecutorServiceTests
   method: testExecute_NotifiesNonRateLimitedTasksOfShutdown
   issue: https://github.com/elastic/elasticsearch/issues/137831
-- class: org.elasticsearch.ingest.SamplingServiceTests
-  method: testMaybeSample
-  issue: https://github.com/elastic/elasticsearch/issues/137871
 - class: org.elasticsearch.xpack.inference.integration.AuthorizationTaskExecutorMultipleNodesIT
   method: testAuthorizationTaskGetsRelocatedToAnotherNode_WhenTheNodeThatIsRunningItShutsDown
   issue: https://github.com/elastic/elasticsearch/issues/137911

--- a/server/src/test/java/org/elasticsearch/ingest/SamplingServiceTests.java
+++ b/server/src/test/java/org/elasticsearch/ingest/SamplingServiceTests.java
@@ -71,7 +71,7 @@ public class SamplingServiceTests extends ESTestCase {
             .putCustom(
                 SamplingMetadata.TYPE,
                 new SamplingMetadata(
-                    Map.of(indexName, new SamplingConfiguration(1.0, maxSize, ByteSizeValue.ofKb(100), TimeValue.timeValueDays(3), null))
+                    Map.of(indexName, new SamplingConfiguration(1.0, maxSize, ByteSizeValue.ofKb(500), TimeValue.timeValueDays(3), null))
                 )
             );
         projectMetadata = projectBuilder.build();

